spec/models/commit_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Commit, type: :model do
+  context 'columns' do
+    it { is_expected.to have_column(:created_at, type: :datetime) }
+    it { is_expected.to have_column(:updated_at, type: :datetime) }
+    it { is_expected.to have_column(:repository_id, type: :integer) }
+    it { is_expected.to have_column(:author_id, type: :integer) }
+    it { is_expected.to have_column(:editor_id, type: :integer) }
+    it { is_expected.to have_column(:author_name, type: :string) }
+    it { is_expected.to have_column(:editor_name, type: :string) }
+    it { is_expected.to have_column(:authored_at, type: :datetime) }
+    it { is_expected.to have_column(:edited_at, type: :datetime) }
+    it { is_expected.to have_column(:shasum, type: :string) }
+  end
```

Comment:
You could add tests for the associations (adding a file, setting an author/editor). I know, these are simple Sequel-defined methods that we would test, but I think we should be better safe than sorry.

Also, corresponding tests for the FileVersion would be great. And tests in Repository (e.g. adding a commit). We should test every possibility that the new code introduces.